src/core/Akka.NodeTestRunner/Program.cs

```diff
@@ -58,6 +73,9 @@ static void Main(string[] args)
                                 specFail.FailureStackTraces.Add(innerEx.StackTrace);
                             }
                             Console.WriteLine(specFail);
+
+                            //make sure message is send over the wire
+                            FlushLogMessages();
```

Comment:
Helper method used to prevent the process from exiting until all messages have been flushed onto the network.